common/src/main/java/com/microsoft/identity/common/internal/broker/MicrosoftAuthClient.java

```diff
@@ -98,6 +98,20 @@ public MicrosoftAuthClient(@NonNull final Context context,
 
                 return bundle;
 
+            case MSAL_GET_INTENT_FOR_ACCOUNT_TRANSFER_INTERACTIVE_REQUEST:
+                final Intent accountTransferIntent = microsoftAuthService.getIntentForAccountTransferInteractiveRequest();
+                final Bundle accountTransferBundle = accountTransferIntent.getExtras();
+
+                //older brokers (pre-ContentProvider) are ONLY sending these values in the intent itself.
```

Comment:
this msg could be confusing - since this operation is just added (it's not pre-ContentProvider)

you actually can just expose 
GetBundleForAccountTransferInteractiveRequestMsalBrokerOperation
on the Broker side.

and return the resulted bundle as is. no need to do extra conversion here.

Reply:
No longer applicable